### PR TITLE
Reboot

### PR DESCRIPTION
**Pull request by judyfong**
Delete the s5 directory except .gitignore
Create directory called s5_base
Add the run.sh files and the accompanying files to prep the data directories.
Mention that s5_base would be the base recipe for all the associated samromur recipes.

### CODE REVIEW

File: s5_base/local/utils.sh [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+undwht='\e[4;37m' # White
+bakblk='\e[40m'   # Black - Background
+bakred='\e[41m'   # Red
+bakgrn='\e[42m'   # Green
+bakylw='\e[43m'   # Yellow
+bakblu='\e[44m'   # Blue
+bakpur='\e[45m'   # Purple
+bakcyn='\e[46m'   # Cyan
+bakwht='\e[47m'   # White
+txtrst='\e[0m'    # Text Reset
```
Comment (Daviderikmollberg) [inline s5_base/local/utils.sh:34]: What are all these colors for?

File: s5_base/local/make_samromur.sh [hunk 2]
```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+# Copyright 2020 Reykjavik University (Judy Fong - judyfong@ru.is)
+# Apache 2.0.
+#
+# This script prepares the samromur dataset for kaldi for training, test, or
+# eval datasets. The new directory have utt2spk,spk2utt,text, wav.scp
+
```
Comment (Daviderikmollberg) [inline s5_base/local/make_samromur.sh]: Just a small comment. make_samromur is not the most descriptive name. 

I would suggest something like samromur_data_prep.

File: s5_base/run.sh [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+if [ $stage -le 4 ]; then
+  echo "Create 3gram language model from kenlm"
+  cat data/train/text | cut -d' ' -f2- | sed -e \
+ 's/[,.?!]//g' > data/text_lm_training.txt
+
+  mkdir -p data/lang_3gram
+
+  # language  modeling files are typically in data/lang_Xgram
+  utils/slurm.pl --mem 16G logs/make_LM_3gsmall.log local/make_LM.sh --order 3 \
```
Comment (Daviderikmollberg) [inline s5_base/run.sh]: the memory allocation should be in cmd

Comment (Daviderikmollberg) [reply s5_base/run.sh]: And another similar fix is ofcourse use the num_jobs flag for --nj. But this is both something that can just be done later.

**Review by Daviderikmollberg — APPROVED** [review]

File: s5_base/local/utils.sh (hunk 1 above)
Comment (judyfong) [reply s5_base/local/utils.sh:34]: I'm actually not sure. Maybe I can remove them. @ingarun Are these colors needed? They were from your local/utils.sh file.

File: s5_base/run.sh (hunk 3 above)
Comment (judyfong) [reply s5_base/run.sh]: I've moved memory to cmd but I'm told that utils/slurm.pl doesn't take in --nj so I need to keep num_jobs currently.

**Pull request merged**